src/js/components/Select/doc.js

```diff
@@ -0,0 +1,60 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { a11yTitlePropType, backgroundPropType } from '../utils';
+
+export default (Select) => {
+  const DocumentedSelect = describe(Select)
+    .description(
+      'An select-like field with optional search capability.'
+    ).usage(
+      `import { Select } from 'grommet';
+      <Select/>`
+    );
+
+  DocumentedSelect.propTypes = {
+    a11yTitle: a11yTitlePropType,
+    background: backgroundPropType,
+    basis: PropTypes.string.description(
+      'Size of the options container inside the Select drop.'
+    ),
+    children: PropTypes.func.description(
+      'Function that will be called when each option is rendered.'
+    ),
+    name: PropTypes.string.description(
```

Comment:
Can we just include this in a `...rest` and not need to document it?

Reply:
we can definitely do this. but I'm using the name inside select

https://github.com/grommet/grommet/pull/1686/files#diff-e73ec96fafca62459d0898bace1bd91fR151